src/components/Modal/Modal.tsx

```diff
@@ -11,39 +11,61 @@ import Heading from 'components/Heading/Heading';
 interface ModalProps {
   children: ComponentChildren;
   className?: any;
-  isOpen: boolean;
+  isOpen: boolean | undefined;
   onClose: () => void;
   title?: string;
   contentWidth?: number | 'boxed';
-  contentVerticalPosition?: number | 'center';
-  contentHorizontalPosition?: number | 'center';
+  contentVerticalPosition?: number | 'center' | 'top' | 'bottom';
+  contentHorizontalPosition?: number | 'center' | 'left' | 'right';
   appendTo: HTMLElement;
 }
 
+function setScrollYProp() {
```

Comment:
where did you get this implementation from?

Reply:
Added the reference